.github/pull_request_template.md

```diff
@@ -14,4 +14,19 @@
 - [ ] Things not downloaded from Red Hat repositories should be mirrored in the Stackable repository and downloaded from there
 - [ ] All packages should have (if available) signatures/hashes verified
 - [ ] Does your change affect an SBOM? Make sure to update all SBOMs
+- [ ] Add an entry to the CHANGELOG.md file
+- [ ] Integration tests ran successfully
 ```
+
+<details>
+<summary>TIP: Running integration tests with a new product image</summary>
+
+The image can be built and uploaded to the kind cluster with the following commands:
+
+```shell
+./build_product_images.py --product <product> --image_version <stackable-image-version>
```

Comment:
This file does not exist

Reply:
https://github.com/stackabletech/docker-images/pull/610

Reply:
thanks both of you, sorry about that!